### PR TITLE
tidy up zen garden

### PR DESCRIPTION
**Pull request by anthony-xiao**
Went through the whole file to make indentation where it seems fit.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                    </nav>
+                </div>
+                <div class="design-archives" id="design-archives">
+                    <h3 class="archives">Archives:</h3>
+                    <nav role="navigation">
+                    <ul>
+                        <li class="next"><a href="/214/page3">Next Designs <span class="indicator">&rsaquo;</span></a></li>
+                        <li class="previous"><a href="/214/page1/"><span class="indicator">&lsaquo;</span> Previous Designs</a></li>
+                        <li class="viewall"><a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden.">
+                        View All Designs</a></li>
```
Comment (bronburgundy) [inline pristine-zen-garden.html:119]: This should be on line above :)

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+                <div class="zen-resources" id="zen-resources">
+                    <h3 class="resources">Resources:</h3>
+                    <ul>
+                        <li class="view-css"><a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">
+                        View This Design&#8217;s <abbr title="Cascading Style Sheets">CSS</abbr></a></li>
+                        <li class="css-resources"><a href="http://www.mezzoblue.com/zengarden/resources/" title="Links to great sites with information on using CSS."><abbr title="Cascading Style Sheets">CSS</abbr> Resources</a></li>
+                        <li class="zen-faq"><a href="http://www.mezzoblue.com/zengarden/faq/" title="A list of Frequently Asked Questions about the Zen Garden."><abbr title="Frequently Asked Questions">FAQ</abbr></a></li>
+                        <li class="zen-submit"><a href="http://www.mezzoblue.com/zengarden/submit/" title="Send in your own CSS file.">
+                        Submit a Design</a></li>
```
Comment (bronburgundy) [inline pristine-zen-garden.html:131]: Same with this one.

**Review by bronburgundy — CHANGES_REQUESTED** [review]
Please @ me when done. Thanks :)

**Comment by anthony-xiao** [comment]
@bronburgundy thanks for the suggestion. definitely missed by me there. Changes are now merged

**Comment by bronburgundy** [comment]
@anthony-xiao I think I'm being a bit slow... but how do I view your changes to sign off on it?

**Comment by anthony-xiao** [comment]
I believe once i have merged it (its done so you are good to go) you will delete the tidying branch then your DONE!

**Pull request merged**